designer/server/src/common/helpers/auth/azure-oidc.js

```diff
@@ -140,5 +137,5 @@ export const azureOidcNoop = {
  * @typedef {import('@hapi/hapi').UserCredentials} UserCredentials - User credentials
  * @typedef {import('oidc-client-ts').OidcMetadata} OidcMetadata - OpenID Connect (OIDC) metadata
  * @typedef {import('oidc-client-ts').SigninResponse} SigninResponse - Provider sign in artifacts
- * @typedef {import('oidc-client-ts').UserProfile & { groups?: string[] }} UserProfile - User profile
+ * @typedef {import('oidc-client-ts').UserProfile & { groups?: string[], unique_name?: string }} UserProfile - User profile
```

Comment:
Adds types to support the [`unique_name` fallback](https://github.com/DEFRA/forms-designer/pull/181/files#r1608133985)